### ISSUE TITLE
Difference between disp_to_depth and colormapped depth image

### ISSUE THREAD

**Issue by space03** [nianticlabs/monodepth2#207]
Thank you for sharing excellent works. 

I'm trying to train and test on my own dataset which is indoor environment.
I have a question about the code.

i'm using  test_simple.py for testing.
In test_simple.py code, #Saving numpy file part and #Saving colormapped depth image part.

When i tested both lines, there was a difference in the results.

1. #Saving numpy file
scaled_disp, scaled_depth = disp_to_depth(disp, 0.1, 100)
np.save(name_dest_npy, scaled_disp.cpu().numpy())
scaled_depth= scaled_depth.cpu()[:, 0].numpy()
img = ax.imshow(scaled_depth[0], cmap=plt.get_cmap('magma'))
![image](https://user-images.githubusercontent.com/41726365/86549550-ea893f80-bf7a-11ea-923a-74857bbe2982.png)

2. #Saving colormapped depth image
disp_resized = torch.nn.functional.interpolate(
	disp, (480, 640), mode="bilinear", align_corners=False)
disp_resized_np = disp_resized.squeeze().cpu().numpy()
vmax = np.percentile(disp_resized_np, 95)
normalizer = mpl.colors.Normalize(vmin=disp_resized_np.min(), vmax=vmax)
mapper = cm.ScalarMappable(norm=normalizer, cmap='magma') #magma or jet
colormapped_im = (mapper.to_rgba(disp_resized_np)[:, :, :3] * 255).astype(np.uint8)
img = ax.imshow(colormapped_im)
![image](https://user-images.githubusercontent.com/41726365/86549565-f2e17a80-bf7a-11ea-8175-30d49d004abe.png)

Why do these two methods make a difference?

I want to compare with the real depth data(realsense or other method).
So what should i use between above two?

**Comment by mrharicot** [comment]
Hi,

`disp` is the "raw" output of the network and is coming from a sigmoid layer so is in [0, 1]
`scaled_disp` is `disp` after being scaled and offset such that its inverse `depth` is bound between 0.1 and 100 units.
https://github.com/nianticlabs/monodepth2/blob/13200ab2f29f2f10dec3aa5db29c32a23e29d376/layers.py#L16-L25
`depth` isn't in meters, it is only up to scale, but is generally the value you want to use.
See [this previous issue](https://github.com/nianticlabs/monodepth2/issues/159#issuecomment-599712590) for more information.

I hope this helps!

**Comment by space03** [comment]
Thank you for fast reply.

I saw before what you mentioned. 
> See this previous issue for more information.

i understand the meaning of disp_to_depth.
anyway, what is the **meaning** of colormapped depth image?

why is different from the image from disp?

**Comment by mrharicot** [comment]
Hi,
You used `depth` in 1 and `disp` in 2, they are the inverse of each other.
